logs/error.log

```diff
@@ -0,0 +1,4 @@
+2019-03-18 22:16:09,931 WARNING: 404 Error Resource Not Found [in /Users/tresor/Desktop/SolarSENSE/SolarSENSE/app/routes.py:314]
```

Comment:
Make sure to remove these error logs.

Reply:
These have been removed and added to the gitignore file